cumulus/README.md

```diff
@@ -1,13 +1,265 @@
+relay chain node in-process. Even though they lack the majority of normal Polkadot subsystems, they
+will still need to connect directly to the relay chain network.
+##### Example command
+```bash
+polkadot-parachain \
+	--chain parachain-chainspec.json \
+	--tmp \
+	--relay-chain-rpc-urls \
+		"ws://relaychain-rpc-endpoint:9944" \
+		"ws://relaychain-rpc-endpoint-backup:9944" \
+	-- \
+	--chain relaychain-chainspec.json
+```
+
+#### Relay Chain Light Client
+An internal relay chain light client provides a fast and lightweight approach for connecting to the relay chain network.
+It provides relay chain notifications and facilitates runtime calls.
+
+To specify which chain the light client should connect to, users need to supply a relay chain chain-spec as part of the relay chain arguments.
+
+**Note:** At this time, any parachain nodes using this feature will still spawn a significantly cut-down
+relay chain node in-process. Even though they lack the majority of normal Polkadot subsystems, they
+will still need to connect directly to the relay chain network.
+
+##### Example command
+```bash
+polkadot-parachain \
+	--chain parachain-chainspec.json \
+	--tmp \
+	--relay-chain-light-client \
+	-- \
+	--chain relaychain-chainspec.json
+```
+
+## Installation and Setup
+Before building Cumulus SDK based nodes / runtimes prepare your environment by following Substrate
+[installation instructions](https://docs.substrate.io/main-docs/install/).
+
+To launch a local network, you can use [zombienet](https://github.com/paritytech/zombienet) for
+quick setup and experimentation or follow the [manual setup](#manual-setup).
+
+### Zombienet
+We use Zombienet to spin up networks for integration tests and local networks.
+Follow [these installation steps](https://github.com/paritytech/zombienet#requirements-by-provider)
+to set it up on your machine. A simple network specification with two relay chain nodes and one collator is
+located at [zombienet/examples/small_network.toml](zombienet/examples/small_network.toml).
+
+
+#### Which provider should I use?
+Zombienet offers multiple providers to run networks. Choose the one that best fits your needs:
+- **Podman:** Choose this if you want to spin up a network quick and easy.
+- **Native:** Choose this if you want to develop and deploy your changes. Requires compilation
+of the binaries.
+- **Kubernetes:** Choose this for advanced use-cases or running on cloud-infrastructure.
+
+#### How to run
+To run the example network, use the following commands:
+
+```bash
+# Podman provider
+zombienet --provider podman spawn ./zombienet/examples/small_network.toml
+
+# Native provider, assumes polkadot and polkadot-parachains binary in $PATH
+zombienet --provider native spawn ./zombienet/examples/small_network.toml
+```
+
+### Manual Setup
+#### Launch the Relay Chain
+
+```bash
+# Clone
+git clone https://github.com/paritytech/polkadot-sdk
+cd polkadot-sdk/polkadot
```

Comment:
I don't think this `cd` should be here. `cargo b ...` works fine in the root, and also the `target` lives in the root so this breaks the following commands prefixed with `./target` 



```suggestion
```